script.js

```diff
@@ -34,7 +34,6 @@ jQuery(function () {
             ;
 
         })
-        .css('cursor', 'pointer')
```

Comment:
I (wrongly) assumed this to be browser-default, but it appears to be only part of the DokuWiki basic styles. Good enough? Otherwise, I'd gladly add it back.